Isabel Chen Exercise 4

ichen2919 opened the pull request:
I still need to include the texts. I'm thinking about having words
trailing behind the cursor.

I want the butterfly to become pixelated or something different when the
mouse is pressed, but I can't figure out how to do that.

ichen2919 committed:
Isabel Chen Exercise 4

I still need to include the texts. I'm thinking about having words
trailing behind the cursor.

I want the butterfly to become pixelated or something different when the
mouse is pressed, but I can't figure out how to do that.

LadyK commented:
Hi Isabel,

It's coming along. Check out the Shiffman text with regard to moving pixels. As well as the assigned chapters in Generative Design for example code of pixelation. Also, you are using an older version of P5 which may cause issues with what you are trying to do. I've outlined this in several emails since our last class.

I look forward to seeing this develop more and to include text.

ichen2919 committed:
Isabel Chen's Exercise 4

There are still some things I cannot figure out with the text; mainly
about the font not being defined.

I tried multiple codes for pixelizing the image but none was able to
work, so I simplified the change, though I still would like to know what
exactly went wrong in the code i put in.
